app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt

```diff
@@ -15,4 +44,116 @@ fun TaskScreen(
     task: Task
 ) {
 
+}
+
+@Composable
+fun SportTaskForm() {
+    var supportsDistanceMetrics by remember { mutableStateOf(false) }
+    var title by remember { mutableStateOf("") }
+    var duration by remember { mutableStateOf("") }
+    var startDate by remember { mutableStateOf(Date()) }
+
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = Color.White)
+    ) {
+        Column(
+            modifier = Modifier
+                .padding(16.dp)
+                .fillMaxSize(),
+            verticalArrangement = Arrangement.Center,
+            //horizontalAlignment = Alignment.CenterHorizontally
+        ){
```

Comment:
Line :67 requires a single space before curly brackets. It should automatically be generated by using auto-formatting of your IDE. By default, the keybind should be `Ctrl+Alt+L`.